app/src/main/java/fr/free/nrw/commons/bookmarks/locations/BookmarkLocationsDao.java

```diff
@@ -189,6 +191,7 @@ private ContentValues toContentValues(Place bookmarkLocation) {
         public static final String TABLE_NAME = "bookmarksLocations";
 
         static final String COLUMN_NAME = "location_name";
+        static final String COLUMN_LANGUAGE = "location_lang";
```

Comment:
Ah, one last thing, how about making the column name `location_language` also?
Other column names are explicit, for instance `location_description` or `location_wikipedia_link`.
(by the way `lat` and `long` are exceptions and not examples to follow, sorry for the inconsistency)

Reply:
Ups, I merged it without seeing this :(

Reply:
should i open a new pr for this one ?

Reply:
I think so, if you do I will merge :)

Reply:
Done.